lib/src/downloader.dart

```diff
@@ -222,7 +222,9 @@ class FlutterDownloader {
             filename: item['file_name'] as String?,
             savedDir: item['saved_dir'] as String,
             timeCreated: item['time_created'] as int,
-            allowCellular: item['allow_cellular'] as bool,
+
+            // allowCellular field is true by default (similar to enqueue())
+            allowCellular: (item['allow_cellular'] as bool?) ?? true,
```

Comment:
@lucaantonelli I feel like this may also need to be applied to `loadTasksWithRawQuery`?